signer/src/signer/zone.c

```diff
@@ -177,48 +182,97 @@ zone_load_signconf(zone_type* zone, signconf_type** new_signconf)
     return status;
 }
 
+int
+sched_task_comparetype(task_type* task, task_id other, task_id* sequence)
```

Comment:
I do not like the sched_task_comparetype function at all...  It should be removed, by not having a strict sequence of tasks in the signer.